examples/va/nfv/ovs-dpdk/control-plane.md

```diff
@@ -0,0 +1,53 @@
+# Configuring networking and deploy the OpenStack control plane
+
+## Assumptions
+
+- A storage class called `local-storage` should already exist.
+
+## Initialize
+
+Switch to the "openstack" namespace
+```
+oc project openstack
+```
+Change to the nfv/ovs-dpdk directory
+```
+cd architecture/examples/va/nfv/ovs-dpdk
+```
+Edit the [values.yaml](values.yaml) and
+[service-values.yaml](service-values.yaml) files to suit 
+your environment.
+```
+vi values.yaml
+vi service-values.yaml
+```
+Alternatively use your own copies of those files and edit
+[kustomization.yaml](kustomization.yaml) to use those copies.
+```
+resources:
+  - values-ci-framework.yaml
+  - service-values-ci-framework.yaml
+```
+
+Generate the control-plane and networking CRs.
```

Comment:
This is not correct as NNCP has been split out into a separate operation. 

https://github.com/openstack-k8s-operators/architecture/commit/e313fb6320607c6d673b3c5878e2c02ab44a8bcf

Please rebase the branch you are submitting this on against main as it is 18 commits behind. 

https://github.com/Jaganathancse/architecture/tree/ovs_dpdk_va_changes

Once you rebase apply the missing steps about nncp to your new control-plane section.